hack/manifest-gen/Makefile

```
@@ -0,0 +1,23 @@
+RESTRICTED_NS:=elastic-system
+DOCKER_IMAGE:=docker.elastic.co/eck/manifest-gen:$(shell cat ../../VERSION)
```

Nit: add spaces and align values.
```suggestion
RESTRICTED_NS := elastic-system
DOCKER_IMAGE  := docker.elastic.co/eck/manifest-gen:$(shell cat ../../VERSION)
```